pkg/manager/index/service/indexer_test.go

```diff
@@ -123,7 +135,7 @@ func Test_index_Start(t *testing.T) {
 		saveIndexTargetAddrCh  chan string
 		schMap                 sync.Map
 		concurrency            int
-		indexInfos             indexInfos
+		indexInfos             func() indexInfos
```

Comment:
It may create the same error at another please :sweat_smile:

Reply:
sorry I can't understand, can you tell me what is `at another` means?